### PR TITLE
Final features for Sprint 1

### PR DESCRIPTION
**Pull request by mk4111**
Adds the last features of Sprint 1

### CODE REVIEW

**Comment by nelsonic** [comment]
@mk4111 please merge master into this branch to reduce the number of commits in the PR.
and/or resolve merge conflicts so I can review. thanks.

**Comment by JMurphyWeb** [comment]
@mk4111 i will merge master into this branch now. Let you know when im done

**Comment by nelsonic** [comment]
@mk4111 pls assign when ready for review. 👍

**Comment by mk4111** [comment]
Hi @nelsonic! Thank you for your message - Jack's just working on getting Heroku working for this. We should then be ready to merge :) Will assign when it's ready!

**Comment by JMurphyWeb** [comment]
@nelsonic i am working off a different branch for deployment PR. There are a few bits still to write, and login is still failing.

I think getting this PR merged would be good.

File: server/email.js [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
 var email = require('sendemail');
 email.set_template_directory('server/email-templates');
 
-var Hoek = require('hoek');
 var sendEmail = {};
 
 sendEmail.newUser = (person, callback) => {
-  const recipient = Object.assign({}, person, { subject: 'Welcome to CE100!' });
-
+  var recipient = Object.assign({}, person, { subject: 'Welcome to CE100!' });
```
Comment (nelsonic) [inline server/email.js]: note this is not currently _sending_ any email. we need to _ensure_ that its working before we can give it to Alex for testing...

**Comment by nelsonic** [comment]
@JMurphyWeb _ideally_ the splitting of `handlers.js` into its respective files should go into master _first_ else you're creating more _re-work_ for tomorrow. but In the interest of _shipping_. I'm merging this into `mater`.

**Pull request merged**